LICENSE

```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2019, Anaconda, Inc.
+Copyright (c) 2019-2020, Anaconda, Inc.
```

Comment:
Unsure it's necessary to keep moving this? Perhaps @seibert will recall.

Reply:
although counter-intuitive, we were told by our lawyer some years ago not to use date ranges for copyrights.  They basically said the important thing was that the initial date was given, and the current copyright holder.

Reply:
Thanks, I'll revert this change then.